app/experimenter/experiments/migrations/0225_nimbusexperiment_updated.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2023-01-16 21:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0224_analysis_add_basis"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="nimbusexperiment",
+            name="updated",
+            field=models.DateTimeField(auto_now=True),
+        ),
+    ]
```

Comment:
Do you want to pre populate it with the datetime of the most recent changelog?

Reply:
the thing is that the option `auto_now and default` are mutually exclusive. we might not be able to provide a default value.

Reply:
❗ ❗ Added migration file to prepopulate `_updated_date_time` from the changelogs